system-int-test/test/cmr/system_int_test/search/acls/collection.clj

```diff
@@ -315,7 +318,7 @@
 
         ;; only permissioned revisions are returned - including tombstones
         "provider-id all-revisions=true"
-        [coll11-1 coll11-2 coll12-1]
```

Comment:
Why did we change the behavior so that tombstones are not returned? This also makes the comment above the test invalid.